docs/dev/install.rst

```diff
@@ -1,67 +1,108 @@
 Developer Setup
 ===============
-1. `Install Docker and Compose`_.
+The following describes how to set up an instance of the site on your
+computer for development.
 
-2. Clone this repository or your fork_:
+Prerequisites
+-------------
+This guide assumes you have already installed and set up the following:
+
+1. Git_
+2. `Python 2.7`_ and pip_
+3. `Node.js`_ and npm_
```

Comment:
For compiling the `es6` JavaScript files, although those are only used on the hidden translate pages. Eventually if we switch to es6 for the majority of the site's JS (which I think we should over time) we'll need it.